otcextensions/sdk/__init__.py

```diff
@@ -98,17 +122,28 @@
         'endpoint_service_type': 'object',
         'set_endpoint_override': True
     },
+    'plas': {
+        'service_type': 'plas'
+    },
     'rds': {
         'service_type': 'rds',
         # 'additional_headers': {'content-type': 'application/json'},
         'endpoint_service_type': 'rdsv3',
         'append_project_id': True,
     },
+    'smn': {
+        'service_type': 'smn',
+        'append_project_id': True
+    },
     'volume_backup': {
         'service_type': 'volume_backup',
         'append_project_id': True,
         'endpoint_service_type': 'vbs',
     },
+    'waf': {
+        'service_type': 'waf',
+        'append_project_id': True,
+    }
 }
 
```

Comment:
Missing rts, evs, elb, workspace, vpcep and sdrs

Reply:
rts - upstream orchestrate, evs - upstream block_storage, elb - upstream "neutron LB" hybrid, workspace - likely soon deprecated
what is vpcep?

Reply:
sdrs added, vpcep is still missing in the docs. I don't think it make sense to add stubs for services which might/might not be deployed to the platform in months/years. Let's have proxies for everything, what we have now.

Reply:
Agree, we can wait until the VPCEP is released